test/binary.js

```diff
@@ -107,4 +107,47 @@ describe.only('binary', () => {
             it(JSON.stringify(value), () => assert.deepStrictEqual(decode(encode(value)), value));
         }
     });
+
+    describe('writer', () => {
+        describe('dynamic size', ()=>{
+            describe('raw', ()=>{
+                it('on size', ()=>{
+                    const writer = new Writer(7);
```

Comment:
I believe we should specify a minimal size for a `chunkSize`, which is a "max" size of chunk. Otherwise, some values (like 8 bytes BigInt) can't be written using  such a `chunkSize`